lib/tasks/anonymize.rake

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+namespace :anonymize do
+  desc "Anonymizes a production dump."
+  task all: %i(users user_groups admins proposals)
+
+  def with_progress(collection, name:)
+    total = collection.count
+    progressbar = create_progress_bar(total: total)
+
+    puts "Anonymizing #{total} #{name}...\n"
+    skip_logs do
+      collection.find_each do |item|
+        yield(item)
+        progressbar.increment
+      end
+    end
+  end
+
+  def create_progress_bar(total:)
+    ProgressBar.create(
+      progress_mark:  " ",
+      remainder_mark: "\u{FF65}",
+      total: total,
+      format: "%a %e %b\u{15E7}%i %p%% %t"
+    )
+  end
+
+  def skip_logs
+    previous_log_level = ActiveRecord::Base.logger.level
+    ActiveRecord::Base.logger.level = 2
+    yield
+    ActiveRecord::Base.logger.level = previous_log_level
+  end
+
+  task proposals: [:environment] do
+    Decidim::Proposals::ProposalVote.delete_all
+
+    with_progress(Decidim::Proposals::Proposal.all, name: "proposals") do |proposal|
```

Comment:
why do we need to remove votes?

Reply:
One might get a user's ID from the production environment (looking at classes on the views, links, or otherwiser) and figure out its votes.

Reply:
Ok, shouldn't we do the same for other kind of contents? (IE Budgets, Surveys, etc)